vaccine_feed_ingest/runners/ri/arcgis/normalize.py

```diff
@@ -0,0 +1,162 @@
+    # accidental mutation.
+    arcgis = "da57e8c8663048a2a9893c636fef63d0"
+    layer = 0
+
+    return f"{runner}:{site}:{arcgis}_{layer}:{data_id}"
+
+
+def _get_inventory(site: dict) -> Optional[List[schema.Vaccine]]:
+    vaccines_field = site["attributes"]["USER_VaxType"].lower().split(",")
+
+    potentials = {
+        "pfizer-biontech covid-19 vaccine": schema.Vaccine(vaccine="pfizer"),
+        "moderna covid-19 vaccine": schema.Vaccine(vaccine="moderna"),
+        "janssen": schema.Vaccine(vaccine="janssen"),
+        "jjj": schema.Vaccine(vaccine="janssen"),
+        "varies": schema.Vaccine(vaccine="varies"),
+    }
+
+    inventory = []
+
+    for vf in vaccines_field:
+        try:
+            if vf != "-" and vf != "\x08":  # "-" is listed when no data given
+                inventory.append(potentials[vf])
+        except KeyError as e:
+            logger.error("Unexpected vaccine type: %s", e)
+
+    if len(inventory) > 0:
+        return inventory
+
+    return None
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    contacts = []
+    if site["attributes"]["USER_Scheduling_by_Phone"]:
+        sourcePhone = re.sub("[^0-9]", "", site["attributes"]["USER_Scheduling_by_Phone"])
+        if len(sourcePhone) == 11:
+            sourcePhone = sourcePhone[1:]
+        phone = f"({sourcePhone[0:3]}) {sourcePhone[3:6]}-{sourcePhone[6:]}"
+        contacts.append(schema.Contact(phone=phone))
+
+    if site["attributes"]["USER_Link_to_Sign_Up"]:
+        contacts.append(schema.Contact(website=site["attributes"]["USER_Link_to_Sign_Up"]))
+
+    if len(contacts) > 0:
+        return contacts
+
+    return None
+
+
+def _get_notes(site: dict) -> Optional[List[str]]:
+    if site["attributes"]["USER_Eligibility"]:
+        return [site["attributes"]["USER_Eligibility"]]
+
+    return None
+
+
+def _get_normalized_location(site: dict, timestamp: str) -> schema.NormalizedLocation:
+    breakpoint()
+    return schema.NormalizedLocation(
+        id=_get_id(site),
+        name=site["attributes"]["USER_Name"],
+        address=schema.Address(
+            street1=site["attributes"]["USER_Address"],
+            street2=None,
+            city=site["attributes"]["USER_City_Town"],
+            state="RI",
+            zip=site["attributes"]["ZIPCODE"],
+        ),
+        location=schema.LatLng(
+            latitude=site["geometry"]["y"],
+            longitude=site["geometry"]["x"],
+        ),
+        contact=_get_contacts(site),
+        languages=None,
+        opening_dates=None,
+        opening_hours=None,
+        availability=None,
+        inventory=_get_inventory(site),
+        access=None,
+        parent_organization=None,
+        links=None,
+        notes=_get_notes(site),
+        active=None,
+        source=schema.Source(
+            source="arcgis",
+            id=site["attributes"]["OBJECTID"],
+            fetched_from_uri="",  # noqa: E501
```

Comment:
Sorry about not providing this info! The URL is `https://rihealth.maps.arcgis.com/apps/instant/nearby/index.html?appid=a25f35833533498bac3f724f92a84b4e`